specification/eventgrid/data-plane/Microsoft.PolicyInsights/stable/2018-01-01/PolicyInsights.json

```diff
@@ -0,0 +1,113 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Policy events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure Policy events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "PolicyInsightsPolicyStateCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.PolicyInsights.PolicyStateCreated event.",
+      "type": "object",
+      "properties": {
+        "timestamp": {
+          "description": "The time (in UTC) that the resource was scanned by Azure Policy.",
+          "type": "string"
+        },
+        "policyAssignmentId": {
+          "description": "The resource ID of the policy assignment.",
+          "type": "string"
+        },
+        "policyDefinitionId": {
+          "description": "The resource ID of the policy definition.",
+          "type": "string"
+        },
+        "policyDefinitionReferenceId": {
+          "description": "The reference ID for the policy definition inside the initiative definition, if the policy assignment is for an initiative. May be empty.",
+          "type": "string"
+        },
+        "complianceState": {
```

Comment:
Maybe this can have an enum if there are a limited set of known states. "complianceReasonCode" is similar.